docs/baking.md

```diff
@@ -247,6 +247,42 @@ In case you want to use a different alias for the baking account:
 2. update the `BAKER_ADDRESS_ALIAS` by editing the
     `/etc/default/tezos-baking-<network>` file.
 
+### Using a custom chain
+
+In case you want to set up a baking instance on a custom chain instead of relying on mainnet
+or official testnets, you can do so:
+
+1. Create a config file for future custom baking instance:
+  ```bash
+  sudo cp /etc/default/tezos-baking-custom@ /etc/default/tezos-baking-custom@<chain-name>
+  ```
+2. In `/etc/default/tezos-baking-custom@<chain-name>`, provide the path to your custom node
+  config in `CUSTOM_NODE_CONFIG`.
+3. Start custom baking service:
+  ```bash
+  sudo systemctl start tezos-baking-custom@<chain-name>
+  ```
+4. Check that all parts are indeed running:
+  ```bash
+  systemctl status tezos-node-custom@<chain-name>
+  systemctl status tezos-baker-011-pthangz2@custom@<chain-name>.service
```

Comment:
This set of services is likely going to change over time.
However, mentioning abstract `<proto>` instead of the specific protocol is probably a bad idea because 1) custom baking service runs daemons for multiple protocols, and the set of protocols changes over time 2) since ithaca there is no endorser daemon, so users may be confused by lack of it.

Perhaps, it's worth leaving this as is, but we should keep in mind that this piece of doc should be updated each time we add/drop protocol support.